src/services/__tests__/buildGame.test.ts

```
@@ -0,0 +1,530 @@
+      };
+
+      const result = customizeBoard(settingsWithLevels, mockActionsFolder, [], 20); // Larger board for more variety
+
+      // Should only include teasing and denial actions, not edging
+      const actionTitles = result.slice(1, -1).map(tile => tile.title);
+      expect(actionTitles).toContain('Teasing');
+      expect(actionTitles).not.toContain('Edging');
+      
+      // Generate multiple times to check that denial can appear (due to randomness)
+      let foundDenial = false;
+      let allFoundTitles = new Set<string>();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'allFoundTitles' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)